src/openms/source/ANALYSIS/ID/FalseDiscoveryRate.cpp

```diff
@@ -1158,6 +1158,7 @@ namespace OpenMS
           if (pepid.getIdentifier() == identifier)
           {
             higher_score_better = pepid.isHigherScoreBetter();
+            break;
```

Comment:
maybe add some comment how higher_score_better is determined? could be a source of errors and easier to spot with one line of comment

Reply:
Yeah, on the other hand if you have a file with mixed orientations you can trash it anyway. Won't be the only algorithm that fails horribly.

Reply:
Will add it

Reply:
I did not add this yet.